packages/kbn-securitysolution-endpoint-exceptions-common/scripts/openapi_bundle.js

```diff
@@ -27,6 +27,13 @@ const ROOT = resolve(__dirname, '..');
           title: 'Security Solution Endpoint Exceptions API (Elastic Cloud Serverless)',
           description: 'Endpoint Exceptions API allow you to manage Endpoint lists.',
         },
+        tags: [
+          {
+            name: 'Security Solution Endpoint Exceptions API',
+            description:
+              "Endpoint Exceptions API allows you to manage detection rule endpoint exceptions to prevent a rule from generating an alert from incoming events even when the rule's other criteria are met.",
```

Comment:
nit: should value of description be the same in both cases, similarly to https://github.com/elastic/kibana/pull/189621/files#diff-1fbdf4b15cda189b67f964f0f03e3c46bf2bbc85f538334b4c9ed38db86aca43R52-R61 ?

Reply:
Exceptions and Endpoint Exceptions APIs are different domains or rather sub-domains. Despite Endpoint Exceptions API reuses Exceptions API schemas under the hood it represents a different set of interactions with the product. UI displays endpoint exceptions in a different tab.

We didn't have Endpoint Exceptions API documented earlier. IMHO having the same tag's name and description might confuse users. Though I'm curious if there are any concerns with that.

Reply:
No, that's not what I meant.

Description on 28th line is different to description on 34.
Whether in PR I referred, both description are the same in tags and root of object